src/com/page/event/mygrades/mygrades.less

```diff
@@ -0,0 +1,11 @@
+@import "defs.less";
+
+.event-mygraded {
+	padding: 1em;
+
+	& .-warning {
```

Comment:
`& > .-warning {`

Reply:
Needed bit restructuring since there was a `div` in-between, but it was useless so removed so now yes, that rule works: Fixed.